src/main/kotlin/no/fdk/dataservicecatalog/controller/CatalogController.kt

```diff
@@ -0,0 +1,59 @@
+package no.fdk.dataservicecatalog.controller
+
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.JSON_LD
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.N3
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.N_QUADS
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.N_TRIPLES
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.RDF_JSON
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.RDF_XML
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.TRIG
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.TRIX
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.TURTLE
+import org.apache.jena.riot.Lang
+import org.apache.jena.riot.RDFLanguages
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestHeader
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/catalogs", produces = [N3, TURTLE, RDF_XML, RDF_JSON, JSON_LD, TRIX, TRIG, N_QUADS, N_TRIPLES])
```

Comment:
Siden disse serialiseringene ikke blir 1 til 1 lik utgavene i JSON og endepunktene skal være helt åpne så tror jeg vi bør gå bort fra at de har samme path. Kanskje prefikse med /published

dvs:
```suggestion
@RequestMapping("/published/catalogs", produces = [N3, TURTLE, RDF_XML, RDF_JSON, JSON_LD, TRIX, TRIG, N_QUADS, N_TRIPLES])
```

Reply:
men det kommer til å ha litt andre følger. Tenker i tillegg at det vi går for i dette apiet er varianten vi endrer de andre registreringsbackendene til å følge.

mao bør nok ta dette opp med resten av teamet

Reply:
vi har visst tatt opp dette tidligere i forbindelse med `service-catalog`, stryk denne kommentaren. Det er json endepunktene som skal ha /internal som prefix